docs/user-guide/platform-linux.md

```diff
@@ -78,6 +77,14 @@ cd pegasus-frontend-git
 makepkg -si
 ```
 
+!!! info "Steam Deck"
+
+Currently, Steam OS is missing 2 required plugins for gstreamer in order for video playback to work in the standalone version. In order to install them, you must temporarily disable Read Only mode, install them and then re-enable Read Only mode as folows:
+```bash
```

Comment:
The documentation is not exactly Bash-specific, so you can  just write `sh` here

Reply:
Oh and please leave a blank line between this and the text, just in case the documentation parser runs into troubles.